### DISCUSSION TITLE
Is there a way to use Add-Pivot table to provide both Min and Max on a single column?

### DISCUSSION THREAD

**Discussion by smswift** [dfinke/ImportExcel#1616] [Q&A]
Add-PivotTable -PivotTableName "Provider Aggregation" -SourceWorksheet $prescriptions -ExcelPackage $package -PivotRows "Approved By", "Diagnosis Code" -PivotData @{"Daily MME Equivalent*" = "Min"; "Daily MME Equivalent*" = "Max"} -PivotDataToColumn

This fails for obvious reasons.. but I would like my pivot chart to show both min and max of a single column. Is this possible?

**Comment by dfinke** [comment]
hmm, off the top of my head, I don't believe so. that fails with duplicate key?

iirc, in Excel it renames the "column" in the pivot so it can show and keeps the underlying column in the data source.

i don't remember making provisions for that. it may be doable if you `Export-Excel`, create the Pivot Table and then get access to the Excel object model, `-PassThru` for example. Then you can update the Pivot directly.

**Reply by smswift** [reply]
Hello! Thanks for the response! Tool has been super helpful. Yes, it fails due to duplicate key. I'll give that a try.

**Reply by dfinke** [reply]
let me know how it goes. it is a good use case.

would also love to hear more how you are using ImportExcel. saw prescriptions, looked up MME, good stuff.
